docs/UserGuide.md

```diff
@@ -281,7 +289,8 @@ exit
 | Mark all          | ```mark all a/ATTENDANCE_TYPE``` <br> e.g., `mark all a/present`
 | Score             | ```score INDEX cp/SCORE``` <br> e.g., `score 1 cp/5`                                             |
 | Score all         | ```score all cp/SCORE``` <br> e.g., `score all cp/10`
-| Lowest score      | `lowest-score`
+| Lowest score      | `lowest-score`      |
+| Random student    | `random-student`     |
```

Comment:
Perhaps having triple backtick to conform to the other entries in the table

Reply:
Fixed!

Reply:
Thanks!